src/TRestRawSignalEvent.cxx

```diff
@@ -354,6 +354,15 @@ Double_t TRestRawSignalEvent::GetMaxTime() {
     return maxTime;
 }
 
+// Specific variable to recover the T2-T1 distance from TRestRawBiPoToSignalProcess
```

Comment:
The comment should be one line long. I would better write something more generic.

`/// An auxiliar variable that can be used to store temporal values.